pom.xml

```diff
@@ -215,14 +215,21 @@ THE SOFTWARE.
         <version>${slf4jVersion}</version>
       </dependency>
       <dependency>
+        <groupId>commons-logging</groupId>
+        <artifactId>commons-logging</artifactId>
+        <version>1.1.3</version>
+        <scope>provided</scope><!-- by jcl-over-slf4j -->
```

Comment:
it can cause havoc with tests though :-(
perhaps `<scope>none</scope>` if that ~~hack~~ undocumented feature still works with the latest maven versions.